src/funnels/automation/StableSwapper.sol

```diff
@@ -73,14 +73,22 @@ contract StableSwapper {
     }
 
     struct PairConfig {
-        uint32  count;  // The remaining number of times that a src to dst swap can be performed by keepers
-        uint112 lot;    // The amount swapped by keepers from src to dst every hop
-        uint112 reqOut; // The minimum output amount to insist on in the swap form src to dst
+        uint128 count;  // The remaining number of times that a src to dst swap can be performed by keepers
+        uint64  hop;
```

Comment:
Don't forget to add documentation to the new vars (can be done at the end).

Reply:
Done!